h5bench_patterns/h5bench_write_unlimited.c

```diff
@@ -0,0 +1,1152 @@
+/****** Copyright Notice ***
+ *
+ * PIOK - Parallel I/O Kernels - VPIC-IO, VORPAL-IO, and GCRM-IO, Copyright
+ * (c) 2015, The Regents of the University of California, through Lawrence
+ * Berkeley National Laboratory (subject to receipt of any required
+ * approvals from the U.S. Dept. of Energy).  All rights reserved.
+ *
+ * If you have questions about your rights to use or distribute this
+ * software, please contact Berkeley Lab's Innovation & Partnerships Office
+ * at  IPO@lbl.gov.
+ *
+ * NOTICE.  This Software was developed under funding from the U.S.
+ * Department of Energy and the U.S. Government consequently retains
+ * certain rights. As such, the U.S. Government has been granted for itself
+ * and others acting on its behalf a paid-up, nonexclusive, irrevocable,
+ * worldwide license in the Software to reproduce, distribute copies to the
+ * public, prepare derivative works, and perform publicly and display
+ * publicly, and to permit other to do so.
+ *
+ ****************************/
+
+/**
+ *
+ * Email questions to SByna@lbl.gov
+ * Scientific Data Management Research Group
+ * Lawrence Berkeley National Laboratory
+ *
+ */
+
+// Description: This is a simple benchmark based on VPIC's I/O interface
+//		Each process writes a specified number of particles into
+//		a hdf5 output file using only HDF5 calls
+// Author:	Suren Byna <SByna@lbl.gov>
+//		Lawrence Berkeley National Laboratory, Berkeley, CA
+// Created:	in 2011
+// Modified:	01/06/2014 --> Removed all H5Part calls and using HDF5 calls
+//          	02/19/2019 --> Add option to write multiple timesteps of data - Tang
+//
```

Comment:
Remove copyright info, update the author and description of the file and timestamps.